Executive.js

```diff
@@ -166,6 +166,8 @@ class Executive {
 	initPlan() {
 		let [year, season] = document.getElementById("startSemesterSelect").value.split('-').map(Number);
 		let major = document.getElementById("majorSelect").value;
+		let x = document.getElementById("showMajor");
+		x.innerHTML= "Major: " + major;
```

Comment:
x is not a very good variable name. I would just assign the value of innerHTML directly instead of storing the element in a temporary variable.